src/main/java/fi/otavanopisto/kuntaapi/server/integrations/casem/CaseMTranslator.java

```diff
@@ -64,6 +75,39 @@ public String slugify(String title) {
     return Collections.singletonList(localizedValue);
   }
   
+  public FileDef translateFile(PageId kuntaApiPageId, FileId kuntaApiFileId, DownloadMeta meta) {
+    String filename = sanitizeFilename(meta.getFilename());
+    
+    FileDef file = new FileDef();
+    file.setContentType(meta.getContentType());
+    file.setId(kuntaApiFileId.getId());
+    file.setSize(meta.getSize() != null ? meta.getSize().longValue() : null);
+    file.setSlug(slugify(filename));
+    file.setTitle(filename);
+    
+    if (kuntaApiPageId != null) {
+      file.setPageId(kuntaApiPageId.getId());
+    }
+    
+    return file;
+  }
+  
+  private String sanitizeFilename(String filename) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "sanitizeFilename" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)